apiary.apib

```diff
@@ -2182,6 +2184,7 @@ Update an invoice.
                         + product_category_id: `e2314517-3cab-4aa9-8471-450e73449041` (string, optional)
         + invoice_date: `2016-02-04` (string, optional)
         + note: `Some comments about the invoice` (string, optional, nullable)
+        + expected_payment_method (ExpectedPaymentMethod, optional)
```

Comment:
We should make this partially updatable as well, so it should be nullable as well.

Reply:
`nullable` here would mean you are removing the `expected_payment_method` if it exists. But is, it is missing nullable but IMO we should keep optional for the partial update.